### PR TITLE
server: add tso retry number log info.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@siddontang @tiancaiamao PTAL

### CODE REVIEW

File: server/tso.go [hunk 1]
```diff
@@ -147,15 +147,15 @@ func (s *Server) getRespTS() *pdpb.Timestamp {
 	for i := 0; i < maxRetryNum; i++ {
 		current, ok := s.ts.Load().(*atomicObject)
 		if !ok {
-			log.Errorf("we haven't synced timestamp ok, wait and retry")
+			log.Errorf("we haven't synced timestamp ok, wait and retry, retry number %d", i)
```
Comment (ngaut) [inline server/tso.go]: s/number/count/

Comment (qiuyesuifeng) [inline server/tso.go]: Agreed.

**Comment by siddontang** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Comment by ngaut** [comment]
LGMT

**Pull request merged**